src/WebpackChunkHandler.ts

```diff
@@ -2,12 +2,14 @@ import { WebpackChunk } from './WebpackChunk';
 import { ModuleCache } from './ModuleCache';
 import { Module } from './Module';
 import { WebpackCompilation } from './WebpackCompilation';
+import { WebpackStats } from './WebpackStats';
 
 interface WebpackChunkHandler {
   processChunk(
     compilation: WebpackCompilation,
     chunk: WebpackChunk,
-    moduleCache: ModuleCache
+    moduleCache: ModuleCache,
+    stats: WebpackStats | undefined
```

Comment:
In what scenario will the stats be undefined?

Reply:
since we do not need to even make this call for WP 3 & 4, we dont. Note the getStats call in the CompilerHandler is only hit if we are using WP5 (guarded by the existance of the processAssets hook). So essentially it is present whenever it is used (WP5) and undefined when it is not (WP<5)